src/meta/src/barrier/checkpoint/control.rs

```diff
@@ -981,6 +981,8 @@ impl DatabaseCheckpointControl {
             return Ok(());
         };
 
+        let mut edges = self.state.inflight_graph_info.build_edge(command.as_ref());
```

Comment:
Why do we only draw the edges in checkpoint control? And not earlier, for instance when preparing the stream graph.

Reply:
The stream graph is prepared outside global barrier manager. In this PR, we only draw the logical edges (FragmentRelation) between fragments, and the physical edges (dispatchers) is drew inside global barrier manager.

Building edges depends the inflight actor info, such as actor vnode bitmap and fragment distribution type, which is currently stored in checkpoint control, so in this PR we build the edges here. We may reorganize the code in the future.